src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1398,4 +1399,30 @@ public static void modifyAttributes()
             FORGE_ATTRIBUTES.put(k, newBuilder.build());
         });
     }
+
+    private static final Set<BlockEntity> LOADING_BLOCK_ENTITIES = Sets.newConcurrentHashSet();
```

Comment:
I don't think maintaining a global static set is an acceptable way of doing this.

Reply:
Moved to `Level`